ResearchKit/ActiveTasks/ORKStroopContentView.m

```diff
@@ -40,6 +40,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 UIStackViewAlignment alignment = UILayoutConstraintAxisHorizontal;
 CGFloat labelHeight = 250.0;
 CGFloat labelWidth = 250.0;
+NSArray <NSLayoutConstraint *> *boxConstraints;
```

Comment:
Why is this a global instead of a field member of ORKStroopContentView?

Reply:
Good catch